flybywiresim/aircraft
# fix(a380x/mfd): Disable MFD INIT & DEPARTURE page buttons depending on fms phase

Fixes #[issue_no]

## Summary of Changes

Hides INIT button in the MFD F-PLN page if the FMS is not in the preflight phase.
Disables the departure button in the INIT page if the phase is not preflight.
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):
bruno_pt99
## Testing instructions

1) Insert a city pair on the INIT page, departure button should be clickable.
2) Verify INIT button appears on the F-PLN page
3) Change the FMS to takeoff phase by setting toga thrust. Verify INIT button disappears on the F-PLN page & that the DEPARTURE button on the INIT page becomes disabled. (You can verify the FMS is no longer in preflight phase by when any of the headers of the MFD PERF page becomes green, such as TO for e.g.)

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
